packages/superset-ui-chart/src/query/buildQueryObject.ts

```diff
@@ -12,8 +14,39 @@ function getGranularity(formData: ChartFormData): string {
 // Note the type of the formData argument passed in here is the type of the formData for a
 // specific viz, which is a subtype of the generic formData shared among all viz types.
 export default function buildQueryObject<T extends ChartFormData>(formData: T): QueryObject {
+  const extras = {
+    druid_time_origin: (formData as DruidFormData).druid_time_origin || '',
+    having: (formData as SqlaFormData).having || '',
+    having_druid: (formData as DruidFormData).having_druid || '',
+    time_grain_sqla: (formData as SqlaFormData).time_grain_sqla || '',
+    where: formData.where || '',
+  };
+
+  const orgGroupby = formData.groupby || [];
```

Comment:
I wasn't sure what `org` meant for these vars initially, could just extract like 

```javascript
const { columns = [], groupby = [] } = formData;

// OR
const groupbySet = new Set([ ...formData.columns, ...formData.groupby ]);
```